packages/@react-native-windows/telemetry/src/utils/sanitizeUtils.ts

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ * @format
+ */
+
+import path from 'path';
+
+import {NpmPackagesWeTrack} from '../telemetry';
+
+const nodeModules = '\\node_modules\\';
+
+const knownEnvironmentVariablePaths = [
+  'AppData',
+  'LocalAppData',
+  'UserProfile',
+];
+
+/**
+ * Gets an anonymized version of the given path, suitable for Telemetry.
+ * @param filepath The path to anonymize.
+ * @param projectRoot Optional root path for the project. Defaults to process.cwd().
+ * @returns The anonymized path.
+ */
+export function getAnonymizedPath(
+  filepath: string,
+  projectRoot?: string,
+): string {
+  projectRoot = (projectRoot ?? process.cwd())
+    .replace(/\//g, '\\')
+    .toLowerCase();
+  filepath = filepath.replace(/\//g, '\\');
+
+  if (filepath.toLowerCase().startsWith(projectRoot)) {
+    // We are under the projectRoot
+    const ext = path.extname(filepath);
+    const rest = filepath.slice(projectRoot.length);
+    if (rest.toLowerCase().startsWith('\\windows\\')) {
+      // We are under the windows path, anonymize with [windows]
+      return `[windows]\\???${ext}(${filepath.length})`;
+    } else if (rest.toLowerCase().startsWith(nodeModules)) {
+      // We are under the node_modules path
+      for (const trackedNpmPackage of NpmPackagesWeTrack) {
+        if (
+          rest
+            .toLowerCase()
+            .startsWith(nodeModules + trackedNpmPackage.replace(/\//g, '\\'))
+        ) {
```

Comment:
this will capture things like react* - need to make sure we either add slashes to all entries in NpmPackagesWeTrack, or better yet add a slash here to force a strict match

Reply:
fixed